src/main/java/io/vertx/core/http/HttpServerResponse.java

```diff
@@ -262,7 +262,40 @@
    * @return a reference to this, so the API can be used fluently
    */
   @Fluent
-  HttpServerResponse sendFile(String filename);
+  default HttpServerResponse sendFile(String filename) {
+    return sendFile(filename, 0);
+  }
+
+  /**
+   * Ask the OS to stream a file as specified by {@code filename} directly
+   * from disk to the outgoing connection, bypassing userspace altogether
+   * (where supported by the underlying operating system.
+   * This is a very efficient way to serve files.<p>
+   * The actual serve is asynchronous and may not complete until some time after this method has returned.
+   *
+   * @param filename  path to the file to serve
+   * @param offset offset to start serving from
+   * @return a reference to this, so the API can be used fluently
+   */
+  @Fluent
+  default HttpServerResponse sendFile(String filename, long offset) {
+    return sendFile(filename, offset, Long.MAX_VALUE);
+  }
+
+  /**
+   * Ask the OS to stream a file as specified by {@code filename} directly
```

Comment:
When creating an overloaded method, instead of duplicating the javadoc, it's preferable to use {@link} to refer to the version with the full javadoc and just explain the differences.